### ISSUE TITLE
Get rid of the generated READMEs

### ISSUE THREAD

**Issue by chasenlehara** [donejs/developer-relations#12]
Repos like [can-component](https://github.com/canjs/can-component) shouldn’t have their READMEs generated. Instead, they should provide basic info about installing, building, finding the docs, etc.

**Comment by andrejewski** [comment]
As a part of this, we should guide people from the Github repo to the API docs in a consistent way. To accomplish that, I propose the Canjs badge:

[![View Canjs documentation](http://svgshare.com/i/2Bm.svg)](https://canjs.com/)

Which will link to the relevant docs.

**Comment by frank-dspeed** [comment]
@andrejewski i am happy with the CANjs badge it looks pritty nice but i think it don't points out that its linking to documentation maybe prepare something with that and on the right side the word documentation

**Comment by chasenlehara** [comment]
Some additional comments related to this issue’s original description:

- I’ve set up the [can-param](https://github.com/canjs/can-param) repo as an example of what I think is an ideal goal
- It looks like the generated part in the READMEs is the entire `## API` section and the list above it; that should be fairly easy to replace with a [Documentation](https://github.com/canjs/can-param/blob/master/README.md#documentation) section with a link to the docs on canjs.com
- The above transformation can probably be written as a codemod and run with [Landscaper](https://github.com/bitovi/landscaper)

I think the above is what’s required for this issue.

Additionally, I think it would be nice to have the READMEs have the same structure as the [can-param README](https://github.com/canjs/can-param/blob/master/README.md), with:

- Sauce Labs and badges under the package title
- The short description
- Documentation, Changelog, Contributing, & License sections

Here are the badges that I think we should standardize on (from a couple projects to show them all; not everything has AppVeyor, Coveralls, etc.):

[![Join the chat at https://gitter.im/canjs/canjs](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/canjs/canjs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![License: MIT](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/canjs/can-param/blob/master/LICENSE.md) [![npm version](https://badge.fury.io/js/can-param.svg)](https://www.npmjs.com/package/can-param) [![Travis build status](https://travis-ci.org/canjs/can-param.svg?branch=master)](https://travis-ci.org/canjs/can-param) [![AppVeyor build status](https://ci.appveyor.com/api/projects/status/github/canjs/can-zone?branch=master&svg=true)](https://ci.appveyor.com/project/matthewp/can-zone) [![Coverage status](https://coveralls.io/repos/github/donejs/donejs/badge.svg?branch=master)](https://coveralls.io/github/donejs/donejs?branch=master) [![Greenkeeper badge](https://badges.greenkeeper.io/donejs/cli.svg)](https://greenkeeper.io/)

It looks like all of the generated READMEs have the same format (title, TOC, API, & Contributing), so I think it’d actually be easiest to use the [can-param README](https://github.com/canjs/can-param/blob/master/README.md) as a [template](https://gist.githubusercontent.com/chasenlehara/1be1c8031ad4a0468cd77bc1f1995218/raw/c6f9c62d419ad3930ac14f73c27dff0147409697/README.md) for replacing each _entire_ generated README, then add a [CONTRIBUTING.md](https://github.com/canjs/can-param/blob/master/CONTRIBUTING.md) file as well.

So, in summary: at a minimum the `## API` section and the list above it need to be replaced in the generated READMEs, but I think it’d be easier to just replace the entire contents of the README.md file and add a CONTRIBUTING.md file. I think this will work for most repos that have generated READMEs, and if it’s not perfect for all of them then we can fix it when reviewing the PRs Landscaper makes.
